source/css/sass/patterns/molecules/teaser-list.scss

```diff
@@ -0,0 +1,18 @@
+@import "../../variables/color";
+@import "../../mixins/decorations";
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+
+.teaser-list__title {
+  @include block-spacing();
+  @include label-content-typography();
+  @include set-font-size-and-line-height(14px);
```

Comment:
Rather than setting explicit font sizes based on the typographical scale used in the design, we now use the modular scale directly.

The scale we use is 1.125 per level (also known as 8:9 ratio, or major second). The excellent online [modular scale](https://www.modularscale.com/?1&em&1.125) site makes it easy to visualise. On this scale, our default font size of 16px for 1 rem maps to a scale of 1 (see the 1em line on the modular scale), which in the sass we have would be expressed as a call to the [`scale()`](https://github.com/libero/pattern-library/blob/master/source/sass/mixins/_scale.scss#L4) function like this: `scale(1)`.

To use a different point on the scale, pass `scale()` the offset you want. For example, for this line of code we want to set the font size closest to 14px. Looking at the scale, that's the one point on the scale lower than 1, so we want to go down a level on the scale, so we pass -1: `scale(-1)`.

So in this case the line would be:
```
@include set-font-size-and-line-height(scale(-1));
```
Note that due to rounding, the scale might not always be exact in every case, but it's close.